### PR TITLE
update install manifest for aeraki

### PR DESCRIPTION
**Pull request by tanjunchen**
### Pre-Submission Checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added related test cases?
* [ ] Have you modified the related document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

**Comment by zhaohuabing** [comment]
@tanjunchen  why we need this?

**Comment by tanjunchen** [comment]
> @tanjunchen why we need this?

@zhaohuabing  
1. Fix the bug that `.Release.Namespace` does not exist，refer to https://github.com/aeraki-mesh/aeraki/pull/336/files#diff-43173311cbcbb96cd0467f3399e82b42a7ac4bfbda0a5697de6e5260c33389e0L22
2. According to https://github.com/istio/istio/blob/master/manifests/charts/base/templates/clusterrolebinding.yaml#L9 We can add namespace  for Charts yaml files

File: k8s/aeraki.yaml [hunk 1]
```diff
@@ -118,7 +118,7 @@ kind: Role
 metadata:
   labels:
     app: aeraki
-  name: aeraki
+  name: aeraki-${AERAKI_NAMESPACE}
```
Comment (zhaohuabing) [inline k8s/aeraki.yaml:121]: Do we have to change the name?

Comment (tanjunchen) [reply k8s/aeraki.yaml:121]: It is not necessary, but after installing multiple aeraki on the same cluster, this should be modified

Comment (zhaohuabing) [reply k8s/aeraki.yaml:121]: It's ok to use the same name for multiple Aeraki deployments since they're in different namespaces.

Comment (tanjunchen) [reply k8s/aeraki.yaml:121]: Installing aeraki in different namespace, aeraki failed to start.
Refer to the istio document, it is recommended to different clusterrole and clusterrolebinding.

**Comment by zhaohuabing** [comment]
Hold this until we have a clear design how multi-tenants would work.

**Comment by zhaohuabing** [comment]
This PR is related to https://github.com/aeraki-mesh/aeraki/issues/330

**Pull request closed without merging**